src/test/java/org/cactoos/iterable/RangeOfTest.java

```diff
@@ -152,4 +152,50 @@ void shouldNotChangeAfterTraversing() throws Exception {
         ).affirm();
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    void producesChars() {
+        new Assertion<>(
+            "Must produce three ranges",
+            new Joined<>(
+                new RangeOf<>('0', '9', ch -> (char) (ch + 1)),
```

Comment:
@andreoss is the cast really needed here?

Reply:
@victornoel Yes, `+` always procudes `int`. Replaced with `++`

Reply:
@victornoel May be it would be nice to have `RangeOfChars`, `RangeOfInts`, etc. 
``new RangeOfInts<>(1,10), new RangeOfInts(1,100,5) /// increment by 5`

Reply:
@andreoss can you create a ticket so that we discuss it there?